podman_compose.py

```diff
@@ -2080,7 +2080,17 @@ def _parse_compose_file(self):
         container_names_by_service = {}
         self.services = services
         for service_name, service_desc in services.items():
-            replicas = try_int(service_desc.get("deploy", {}).get("replicas"), fallback=1)
+            replicas = try_int(args.scale, fallback=1) if "scale" in args else 1
```

Comment:
I think we should use `None` to signify that no value has been supplied, and then handle the None case at the very end. Right now if explicit 1 value is passed it may be overridden by next steps.

Reply:
I've updated the code to handle the cases:
1. `--scale` from the CLI `up` command
2. `scale` from compose yaml file
3. `{"deploy": {"replicas": <NUM>}` from compose yaml file

Was tempted to use the new fancy match-cases, but didn't to maintain compatibility with pre-Python-2.10 versions.

Let me know if the new code is fine.